package/windows/Dockerfile.agent

```diff
@@ -12,7 +12,7 @@ RUN $URL = 'https://github.com/StefanScherer/docker-cli-builder/releases/downloa
     \
     Write-Host 'Complete.'
 # download wins
-RUN $URL = 'https://github.com/thxCode/wins/releases/download/v0.0.11/wins.exe'; \
+RUN $URL = 'https://github.com/thxCode/wins/releases/download/v0.1.0/wins.exe'; \
```

Comment:
Should this be moved from https://github.com/thxCode/wins to https://github.com/rancher/wins? And where is this hosted/being built?